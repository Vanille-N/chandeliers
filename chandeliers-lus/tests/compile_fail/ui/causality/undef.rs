//! Missing a definition.
//! If rustc is made to catch this the error will be atrocious, so we
//! really want chandeliers to flag it.
chandeliers_lus::decl! {
    node foo() returns (i : int);
    let tel;
}

fn main() {}
