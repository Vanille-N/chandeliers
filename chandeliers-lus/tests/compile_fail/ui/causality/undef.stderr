error: No definition provided for Variable `i` which is required
 --> tests/compile_fail/ui/causality/undef.rs:2:25
  |
2 |     node foo() returns (i : int);
  |                         ^
